tsconfig.test.json

```diff
@@ -2,6 +2,7 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "suppressImplicitAnyIndexErrors": true,
+    "resolveJsonModule": true
```

Comment:
Out of curiosity, what does this do? 👀

Reply:
It allows importing `json` files in typescript files. You cannot fo this for example `import x from 'blabla.json'` by default.